google-workloads-to-champsim/README.md

```diff
@@ -0,0 +1,95 @@
+# Google Workload Traces to Champsim Wrapper Script
+
+## Compile
+Use the following command to compile: 
+
+`g++ google-workload-to-champsim.cpp -lz` 
+
+## Run
+Use the following command to run the executable:
+
+`./a.out <trace>`
+
+`<trace>` should be a google workload trace file in compressed form with `.gz` extension. Google workload traces can be obtained [here](https://dynamorio.org/google_workload_traces.html).
```

Comment:
drmemtrace traces are now in .zip format.  Sample traces are in the https://github.com/DynamoRIO/drmemtrace_samples repository.